src/types/cognitoTypes.ts

```diff
@@ -0,0 +1,25 @@
+export type CognitoConfirmSignUpParams = {
```

Comment:
its relevant for all types in this file - check if there is types you can import from the aws-sdk module instead of declare it by yourself